### ISSUE TITLE
The plugin is still developed?

### ISSUE THREAD

**Issue by ckubs** [travispflanz/wordpress-recent-popular-tags-plugin#1]
Tried to use the plugin, installed it from wordpress.org but it's not working and it's the only one of it's kind to show popular tags from a certain period of time. 
I guess the problem is the compatibility with the latest wordpress iteration, as the settings aren't saved.

**Comment by fastmover** [comment]
Do you have Google hangouts?  I'd like to find out why this is happening.

**Comment by ckubs** [comment]
No, I don't use that :( This plugin is nice though but innability to save the options is bad.
